django_rq/queues.py

```diff
@@ -24,16 +25,17 @@ def get_commit_mode():
     return RQ.get('AUTOCOMMIT', True)
 
 
-def get_queue_class(config):
+def get_queue_class(config={}, queue_class=None):
```

Comment:
Could you please change the default argument to `None`? Using a mutable object (`{}`) as default argument often introduce hard to find bugs.

Reply:
Agreed.